tutorials/scripting/cross_language_scripting.rst

```diff
@@ -171,6 +171,30 @@ If that's impossible, you'll see the following error: ``Invalid call. Nonexisten
     my_csharp_node.PrintArray(["a", "b", "c"]) # a, b, c
     my_csharp_node.PrintArray([1, 2, 3]) # 1, 2, 3
 
+Receiving C# collections from GDScript
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When passing or returning collections from C# methods, types from the `System.Collections` and `System.Collections.Generic` namespaces are not compatible with Godot's marshalling system. For example:
+
+.. code-block:: csharp
+
+    // These methods will not be visible to GDScript, and Godot will throw the "Invalid call" error
+    public List<string> GetStrings() { ... }
+
+To ensure your collections can be used in GDScript, use Godot's collection types from the `Godot.Collections` namespace:
```

Comment:
```suggestion
To ensure your collections can be used in GDScript, use Godot's collection types from the `Godot.Collections` namespace. The types in your collection also must be :ref:`Variant-compatible <c_sharp_variant_compatible_types>`:
```
I'd also mention the variant compatibility requirement. Feel free to reword or move it to somewhere else in the section.